jobs/models.py

```diff
@@ -69,7 +68,81 @@ def set_expiration_date(self):
                 self.expiration_date = self.published_date + timedelta(60)
                 self.save()
 
-
     def __unicode__(self):
         return "{0}, {1}".format(self.title, self.company)
 
+
+class Meetup(models.Model):
+
+    MEETUP = 'MEET'
+    CONFERENCE = 'CONF'
+    WORKSHOP = 'WORK'
+    MEETUP_TYPES = (
+        (MEETUP, 'meetup'),
+        (CONFERENCE, 'conference'),
+        (WORKSHOP, 'workshop'),
+    )
+
+    title = models.CharField(max_length=500)
+    organization = models.ForeignKey(
+        'Company',
+        related_name="meetups",
+        blank=True,
+        null=True,
+    )
+    type = models.CharField(max_length=4, choices=MEETUP_TYPES, default=MEETUP)
+    contact_email = models.EmailField(max_length=254)
+    city = models.CharField(max_length=100)
+    country = CountryField()
+    description = models.TextField(max_length=5000)
+    is_recurring = models.BooleanField(
+        default=False,
+        help_text="Is your meetup recurring?"
+    )
+    meetup_date = models.DateTimeField(
+        null = True,
+        help_text="This stands for a starting date if the meetup is recurring"
+    )
+    reviewer = models.ForeignKey(
```

Comment:
I was thinking a CharField as well as the Boolean, so we can capture the
additional details about the recurrence. What do you think?
On 12 Feb 2015 16:21, "Marysia" <notifications@github.com> wrote:

> In jobs/models.py
> <https://github.com/djangogirlsjobs/djangogirls/pull/7#discussion_r24593558>
> :
>
> > +        null=True,
> > +    )
> > +    type = models.CharField(max_length=4, choices=MEETUP_TYPES, default=MEETUP)
> > +    contact_email = models.EmailField(max_length=254)
> > +    city = models.CharField(max_length=100)
> > +    country = CountryField()
> > +    description = models.TextField(max_length=5000)
> > +    is_recurring = models.BooleanField(
> > +        default=False,
> > +        help_text="Is your meetup recurring?"
> > +    )
> > +    meetup_date = models.DateTimeField(
> > +        null = True,
> > +        help_text="This stands for a starting date if the meetup is recurring"
> > +    )
> > +    reviewer = models.ForeignKey(
>
> Do you mean that we should have a simple CharField for recurrence, so that
> people can describe the sort of recurrence they need instead of the
> BooleanField?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/djangogirlsjobs/djangogirls/pull/7/files#r24593558>.
>